mod-source-record-storage-server/src/main/java/org/folio/rest/impl/SourceStorageBatchImpl.java

```diff
@@ -53,4 +54,26 @@ public void postSourceStorageBatchRecords(RecordCollection entity, Map<String, S
       }
     });
   }
+
+  @Override
+  public void putSourceStorageBatchParsedRecords(ParsedRecordCollection entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(v -> {
+      try {
+        recordService.updateParsedRecords(entity, tenantId)
+                .map((ParsedRecordCollection it) -> {
+                  if (it.getErrorMessages().isEmpty()) {
+                    return (Response) PutSourceStorageBatchParsedRecordsResponse.respond200WithApplicationJson(it);
+                  } else {
+                    LOG.error("Some records were not saved! Here is the list of errors: {}", it.getErrorMessages());
+                    return (Response) PutSourceStorageBatchParsedRecordsResponse.respond500WithApplicationJson(it);
```

Comment:
I disagree with the 500 status here, but will back off if the team agrees that it's an appropriate solution. Let's move forward with the spike https://issues.folio.org/browse/MODSOURCE-63

Reply:
agree